### PR TITLE
misc refactorings

### PR DESCRIPTION
**Pull request by turadg**
## Description

Some refactors that had come up
- https://github.com/Agoric/agoric-sdk/pull/7000#discussion_r1118028462
- https://github.com/Agoric/agoric-sdk/pull/7100#discussion_r1125689349

### CODE REVIEW

File: packages/inter-protocol/src/auction/auctioneer.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ import { makeAuctionBook } from './auctionBook.js';
 import { AuctionState } from './util.js';
 import { makeScheduler } from './scheduler.js';
 import { auctioneerParamTypes } from './params.js';
+import { appendToStoredArray } from '@agoric/store/src/stores/store-utils.js';
```
Comment (erights) [inline packages/inter-protocol/src/auction/auctioneer.js]: How about we make it a direct export from `@agoric/store` and avoid the deep import?

File: packages/zoe/src/contractSupport/ratio.js [hunk 2]
```diff
@@ -373,7 +373,7 @@ export const parseRatio = (
     throw Fail`Invalid numeric data: ${numeric}`;
   }
 
-  const [whole, part = ''] = [match[1], match[2]];
+  const [_, whole, part = ''] = match;
```
Comment (erights) [inline packages/zoe/src/contractSupport/ratio.js:376]: Doh! Thx.

**Review by erights — APPROVED** [review]
LGTM, thanks.

**Review by Chris-Hibbert — APPROVED** [review]

File: packages/inter-protocol/src/auction/auctioneer.js (hunk 1 above)
Comment (turadg) [reply packages/inter-protocol/src/auction/auctioneer.js]: Probably better but not worth another CI spin imo. I expect we'll clean up deep imports after or as part of refactoring the repo.

**Pull request merged**